cmd/makedocs/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"html/template"
```

Comment:
You should probably use `text/template`.
This way, there shouldn't be any weird escaping problems in the output.